src/SCRIPTS/BF/PAGES/pos_osd.lua

```diff
@@ -0,0 +1,240 @@
+        {["name"]="OSD_CORE_TEMPERATURE",        ["address"]=40,  ["val_1"]=91,  ["val_2"]=92, ["name_cli"]="osd_core_temp_pos"},
+        {["name"]="OSD_ANTI_GRAVITY",            ["address"]=41,  ["val_1"]=93,  ["val_2"]=94, ["name_cli"]="osd_anti_gravity_pos"},
+        {["name"]="OSD_G_FORCE",                 ["address"]=42,  ["val_1"]=95,  ["val_2"]=96, ["name_cli"]="osd_g_force_pos"},
+        {["name"]="OSD_MOTOR_DIAG",              ["address"]=43,  ["val_1"]=97,  ["val_2"]=98, ["name_cli"]="osd_motor_diag_pos"},
+        {["name"]="OSD_LOG_STATUS",              ["address"]=44,  ["val_1"]=99, ["val_2"]=100, ["name_cli"]="osd_log_status_pos"},
+        {["name"]="OSD_FLIP_ARROW",              ["address"]=45, ["val_1"]=101, ["val_2"]=102, ["name_cli"]="osd_flip_arrow_pos"},
+        {["name"]="OSD_LINK_QUALITY",            ["address"]=46, ["val_1"]=103, ["val_2"]=104, ["name_cli"]="osd_link_quality_pos"},
+        {["name"]="OSD_FLIGHT_DIST",             ["address"]=47, ["val_1"]=105, ["val_2"]=106, ["name_cli"]="osd_flight_dist_pos"},
+        {["name"]="OSD_STICK_OVERLAY_LEFT",      ["address"]=48, ["val_1"]=107, ["val_2"]=108, ["name_cli"]="osd_stick_overlay_left_pos"},
+        {["name"]="OSD_STICK_OVERLAY_RIGHT",     ["address"]=49, ["val_1"]=109, ["val_2"]=110, ["name_cli"]="osd_stick_overlay_right_pos"},
+        {["name"]="OSD_DISPLAY_NAME",            ["address"]=50, ["val_1"]=111, ["val_2"]=112, ["name_cli"]="osd_display_name_pos"},
+        {["name"]="OSD_ESC_RPM_FREQ",            ["address"]=51, ["val_1"]=113, ["val_2"]=114, ["name_cli"]="osd_esc_rpm_freq_pos"},
+        {["name"]="OSD_RATE_PROFILE_NAME",       ["address"]=52, ["val_1"]=115, ["val_2"]=116, ["name_cli"]="osd_rate_profile_name_pos"},
+        {["name"]="OSD_PID_PROFILE_NAME",        ["address"]=53, ["val_1"]=117, ["val_2"]=118, ["name_cli"]="osd_pid_profile_name_pos"},
+        {["name"]="OSD_PROFILE_NAME",            ["address"]=54, ["val_1"]=119, ["val_2"]=120, ["name_cli"]="osd_profile_name_pos"},
+        {["name"]="OSD_RSSI_DBM_VALUE",          ["address"]=55, ["val_1"]=121, ["val_2"]=122, ["name_cli"]="osd_rssi_dbm_pos"},
+        {["name"]="OSD_RC_CHANNELS",             ["address"]=56, ["val_1"]=123, ["val_2"]=124, ["name_cli"]="osd_rcchannels_pos"},
+        {["name"]="OSD_CAMERA_FRAME",            ["address"]=57, ["val_1"]=125, ["val_2"]=126, ["name_cli"]="osd_camera_frame_pos"},
+        {["name"]="OSD_EFFICIENCY",              ["address"]=58, ["val_1"]=127, ["val_2"]=128, ["name_cli"]="osd_efficiency_pos"},
+        {["name"]="OSD_TOTAL_FLIGHTS",           ["address"]=59, ["val_1"]=129, ["val_2"]=130, ["name_cli"]="osd_total_flights_pos"},
+        {["name"]="OSD_UP_DOWN_REFERENCE",       ["address"]=60, ["val_1"]=131, ["val_2"]=132, ["name_cli"]="osd_up_down_reference_pos"},
+        {["name"]="OSD_TX_UPLINK_POWER",         ["address"]=61, ["val_1"]=133, ["val_2"]=134, ["name_cli"]="osd_link_tx_power_pos"},
```

Comment:
As this is going to be released together with BF 4.4 please also add
```
WH_DRAWN
AUX_VALUE
READY_MODE
SYS_GOGGLE_VOLTAGE
SYS_VTX_VOLTAGE
SYS_BITRATE
SYS_DELAY
SYS_DISTANCE
SYS_LQ
SYS_GOGGLE_DVR
SYS_VTX_DVR
SYS_WARNINGS
SYS_VTX_TEMP
SYS_FAN_SPEED
```

Reply:
where can I find this in betaflight code? I can´t find the defintions anywhere if I search betaflight github.